daffodil-test/src/test/resources/org/apache/daffodil/section12/aligned_data/Aligned_Data.tdml

```diff
@@ -2090,11 +2090,10 @@
   </tdml:document>
   <tdml:infoset>
     <tdml:dfdlInfoset><e10b><varArr><e>11</e><e>4</e><e>14</e><e>9</e><e>11</e></varArr><one>11</one></e10b>
-    </tdml:dfdlInfoset>
+  </tdml:dfdlInfoset>
   </tdml:infoset>
   <tdml:warnings>
-    <tdml:warning>Schema Definition Warning</tdml:warning>
-    <tdml:warning>e is an optional element or a variable-occurrence array and its alignment (2) is not the same as one's alignment (1)</tdml:warning>
+    <tdml:warning>Schema Definition Warning Palceholder</tdml:warning>
```

Comment:
Was this "placeholder" intentional?

Reply:
I originally thought the SDW would have a different message for some reason, but it looks like that is the correct message to receive. I'll revert this.